src/main/java/bob/ui/Ui.java

```diff
@@ -0,0 +1,124 @@
+package bob.ui;
+
+import java.util.Scanner;
+
+import bob.task.Task;
+
+/**
+ * Class responsible for handling Ui
+ */
+public class Ui {
+    private static final String HORITZONAL_LINE = "____________________________________________________________";
```

Comment:
Minor typo in variable name here
```suggestion
    private static final String HORIZONTAL_LINE = "____________________________________________________________";
```